src/actions/geek-lab/config.js

```diff
@@ -0,0 +1,35 @@
+exports.command = 'config';
```

Comment:
not adding any instructions to postintall script yet telling the user to execute
`geek-lab config --env ENV --key KEY --value VALUE` because its not required yet